connection.go

```diff
@@ -302,26 +303,26 @@ func (sc *snowflakeConn) ExecContext(ctx context.Context, query string, args []d
 				updatedRows += count
 			}
 		}
-		logger.Infof("number of updated rows: %#v", updatedRows)
+		logger.WithContext(ctx).Infof("number of updated rows: %#v", updatedRows)
 		return &snowflakeResult{
 			affectedRows: updatedRows,
 			insertID:     -1,
 			queryID:      sc.QueryID,
 		}, nil
 	}
-	logger.Info("DDL")
+	logger.Debug("DDL")
```

Comment:
Should this be `logger.WithContext(ctx).Debug` ?

Reply:
Not sure if we even need this log message. For now I just change it to Debug level. If we don't see any benefit from this log message in the future - providing progress or event info etc. we could remove it.

Reply:
That's good point. If no log is required, it should be removed. I believe this was added for more or less debugging. This doesn't look useful.